Dockerfile

```diff
@@ -130,9 +130,8 @@ RUN --mount=target=/ccache,type=cache \
       ${TENZIR_BUILD_OPTIONS} && \
     cmake --build build --parallel && \
     ctest --test-dir build --output-on-failure --exclude-regex "^tenzir/" && \
-    cmake --build build --target bats && \
```

Comment:
Was this accidentally removed?

Reply:
Yes.

Reply:
"Yes" means it was an accident. I restored it.